mob.go

```diff
@@ -40,7 +40,11 @@ func openCommandFor(c config.Configuration, filepath string) (string, []string)
 	if !c.IsOpenCommandGiven() {
 		return "", []string{}
 	}
-	split := strings.Split(injectCommandWithMessage(c.OpenCommand, filepath), " ")
```

Comment:
I think we should be able to come up with a solution that does not need replacement with an arbitrary placeholder.
E.g. If we split the `OpenCommand` by space first (even before injecting the filepath), and only then inject the filepath into each of the pieces.

Reply:
As I see it, splitting `OpenCommand` before injecting the `filepath` would require a special version of `injectCommandWithMessage` that works on the list of pre-split arguments. Otherwise, the "single or no placeholder"-semantics of `injectCommandWithMessage` could not be enforced here.
If we drop this constraint here and assume that there is exactly one %s in OpenCommand, a simple `strings.ReplaceAll` on each split part (command argument) would do the trick, but that would modify existing behavior and duplicate parts of it.

Both of those solutions don't seem better to me than my proposed solution. Do you prefer one of those, or maybe have a better idea how to solve this?

As I contemplated a better solution, I came to the following insights and ideas:
- The problem here seems to be, that `injectCommandWithMessage` is not aware of the fact, that commands may have arguments that contain spaces
- Technically, this could also affect the other calls of `injectCommandWithMessage`
- A cleaner solution would probably have to model the difference between arguments and "words separated by spaces" explicitly.
- This might need to happen as early as parsing the config, as users might want to use double quotes in the config to mark arguments that contain spaces
- I would not be comfortable doing this kind of remodeling in an unfamiliar code-base (mob.sh) and foreign language (go)
- I would, however, be willing to pair- or ensemble-program this with someone more familiar whit both the codebase and go

Which path forward would you prefer?